src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,207 @@
+package duke;
+
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.Scanner;
+
+/**
+ * <h1>duke.Duke!</h1>
+ * The duke.Duke program implements an application that
+ * helps you keep track of your tasks.
+ *
+ * @author Augustine Kau
+ * @version 0.1
+ * @since 2020-08-18
```

Comment:
The javadocs are quite professional since you added the version as well as your name.